.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
Please add a newline at the end of the file.

Reply:
added